app/src/main/java/com/example/bigowlapp/activity/SearchContactsByPhone.java

```diff
@@ -35,13 +30,16 @@
     private Button btnSearch;
     private final FirebaseFirestore db = FirebaseFirestore.getInstance();
     private static final ArrayList<QueryDocumentSnapshot> qds = new ArrayList<QueryDocumentSnapshot>();
+    private EditText inputNumber;
+    private String number2;
```

Comment:
You should use more descriptive name. I'm not sure what `number2` means. And `inputNumber` which I assume represent a phone number text input shot be like `inputPhoneNumber`